tests/containers/runtimes/runtime_test.py

```
@@ -0,0 +1,39 @@
+        container = WorkbenchContainer(image=runtime_image.name, user=4321, group_add=[0])
+
+        def check_zmq():
+            # ruff: noqa: PLC0415 `import` should be at the top-level of a file
+            import zmq  # pyright: ignore reportMissingImports
+
+            context = zmq.Context()
+            try:
+                socket = context.socket(zmq.PAIR)
+                print("pyzmq imported and socket created successfully")
+            finally:
+                socket.close(0)  # linger=0
```

Pylint (3.3.7)

[error] Using variable 'socket' before assignment

(E0601)